src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -946,9 +960,9 @@ protected virtual void CloseCore(CancellationToken token)
                 try
                 {
                     TimeoutHelper helper = new TimeoutHelper(ServiceDefaults.CloseTimeout);
-                    if (_securityProtocol != null)
+                    if (SecurityProtocol != null)
                     {
-                        _securityProtocol.CloseAsync(false, helper.RemainingTime()); ;
+                        SecurityProtocol.CloseAsync(false, helper.RemainingTime()); ;
                     }
```

Comment:
There are multiple async calls in this method which could cause problems not awaiting them.
Nit: Double semi colon